packages/cloud-config/build.yaml

```diff
@@ -3,6 +3,11 @@ requires:
   category: "system"
   version: ">=0"
 steps:
-- mkdir -p /oem
-- cp -rfv yipfiles/*.yaml /oem
-- chmod -R 600 /oem
\ No newline at end of file
+- mkdir -p /system/oem
+- cp -rfv yipfiles/*.yaml /system/oem
+- chmod -R 600 /system/oem
+{{if .Values.is_opensuse}}
+- sed -i 's/:FLAVOR:/opensuse/g' /system/oem/02_upgrades.yaml
+{{else}}
```

Comment:
I would use `{{else if .Values.is_fedora}}` here to be consistent. Other packages have no default, you must either select fedora or opensuse. So if you do not provide a `--values` with the corresponding flavor the it will just fail.

Reply:
right! fixed it :+1: